src/main/java/me/thegiggitybyte/wumpuslink/WumpusLink.java

```diff
@@ -98,7 +89,7 @@ public void onInitializeServer() {
             var configCommand = CommandManager.literal("config")
                     .requires(Permissions.require("wumpuslink.reload.config", 4))
                     .executes(ctx -> {
-                        initializeConfig();
+                        JsonConfiguration.getUserInstance();
```

Comment:
`JsonConfiguration` has no mechanism to read from the JSON file after the class has been initialized; perhaps add a simple method to `JsonConfiguration` which does this? (see static init for `JsonConfiguration`)

Reply:
Decided to remove reloading, and instead use config command from SleepWarp.